packages/xo-web/src/xo-app/xoa/update/index.js

```diff
@@ -169,7 +169,14 @@ const Updates = decorate([
       },
       toggleState,
       update: () => xoaUpdater.update(),
-      upgrade: () => xoaUpdater.upgrade(),
+      async upgrade() {
+        return this.props.runningJobsExist
```

Comment:
Where does it come from?

Reply:
https://github.com/vatesfr/xen-orchestra/blob/master/packages/xo-web/src/xo-app/xoa/update/index.js#L288

Reply:
We don't need anymore to use `HTML datasets`, I changed this.

Reply:
So, `this.props.runningJobsExist` does not exist, we need to use `this.state.areJobsRunning` instead, and `data-runningJobsExist` was unused, you were right to remove it.

Reply:
What I found weird is that `data-runningJobsExist` still works and I tested this

Reply:
What do you mean by “works”?

Reply:
`this.props.runningJobsExist` is `true` when jobs are running and `false` when it's not the case